examples/server/v1/components.py

```diff
@@ -63,6 +64,7 @@ def __init__(self, name: str):
         )
         self.joint_positions = JointPositions(values=[0, 0, 0, 0, 0, 0])
         self.is_stopped = True
+        self.kinematics = (KinematicsFileFormat.KINEMATICS_FILE_FORMAT_SVA, b"\x00\x01\x02")
```

Comment:
Why an ExampleArm in addition to the ModuleArm?

Reply:
This `ExampleArm` is just another part of the `server` example that includes an example of every component we have. Almost, if not all, the values are arbitrary. The ticket specifically asked for a better arm module, which is what `ModuleArm` is for.